tutorials/cp-starter-ibpm-employeeonboarding-1-setup/cp-starter-ibpm-employeeonboarding-1-setup.md

```diff
@@ -26,25 +26,31 @@ To create service instances for business process management services, you need t
 
 1. In your web browser, open the [SAP Cloud Platform Trial cockpit](https://account.hanatrial.ondemand.com/cockpit).
 
-2. Navigate into the trial subaccount
+2. Click on **Enter Your Trial Account** to navigate into the trial account
 
       !![Open Trial Account](opentrialaccount.png)
 
     >The trial account should contain one `subaccount` and `space`
 
-3. Open **Entitlements** from left panel. You should have the following entitlements in your account.
+3. Click on the subaccount.
 
-    >If not, then use **Configure Environments** to add these entitlements.
+    !![Enter Trial Subaccount](entertrialaccount.png)
 
-    Here is an explanation of these different entitlements:
+4. Click **Entitlements** from left panel to check if you have all the below listed entitlements in your subaccount. These entitlements are needed to run other tutorials of this mission.
 
-    - Connectivity is needed to configure destination
-
-    - Portal and HTML5 Application is needed to access Process Visibility and Workflow applications from Fiori Launchpad
+    |  Entitlement     | Description
+    |  :------------- | :-------------
+    |  Business Rules, Workflow and Process Visibility | These are core services of Intelligent BPM which will be needed to run the next tutorials.
+    |  Portal, Application Runtime, HTML5 Applications | These services are needed to setup and access Process Visibility and Workflow applications from Fiori Launchpad
+    |  Connectivity    | This service is needed to create and configure destinations
 
-    - Business Rules, Workflow and Process Visibility is needed to create service instances of these respective services in order to work with them.
+    - If you do not see these entitlements in your subaccount, then do the following:
+        - click **Configure Entitlements**
```

Comment:
For these four sentences please capitalize and finish with a period.
The bullets are OK.